wagtailnhsukfrontend/templates/wagtailnhsukfrontend/skip_link.html

```diff
@@ -0,0 +1 @@
+<a class="nhsuk-skip-link" href=#{{ href|default:"#maincontent" }}> {{ text|default:"Skip to main content" }} </a>
```

Comment:
`href=#{{ href|default:"#maincontent" }}`

This means the default becomes `href=##maincontent`!

Do you expect people to use the template like this:
`{% include ... with href="#main" %}`
or this?
`{% include ... with href="main" %}`

Reply:
The original intention was to include the hashtag already but in retrospect, I can see why this might cause confusion. Will remove the hashtag and let the user enter it.